vllm/config/scheduler.py

```diff
@@ -181,6 +201,12 @@ def __post_init__(self, is_encoder_decoder: bool) -> None:
                 " prefix caching; disabling both."
             )
 
+        if self.enable_hybrid_chunked_prefill is None:
+            self.enable_hybrid_chunked_prefill = False
```

Comment:
If we immediately set it to `False` can we just set the actual default to `False` instead of `None`?